### PR TITLE
[main.yml] Updated to openwrt-18.06 #16

### LINKED ISSUE #16: Make openwrt 18.06 default
Make openwrt 18.06 the default.

Shouldn't be hard, see these lines:

- https://github.com/ninuxorg/ansible-nnxx/blob/master/chilli.yml#L10-L11
- https://github.com/ninuxorg/ansible-nnxx/blob/master/chilli.yml#L44-L57

### PR DESCRIPTION
**Pull request by EdgeKing810**
Updated 'lede' with openwrt-18.06 and added some more packages. 

Please let me know if I have followed the guidelines well this time.

Closes #16

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -37,15 +37,19 @@ openwisp2fw_source_feeds:
...
-      branch: lede-17.01
+      branch: openwrt-18.06
     - method: src-git
       name: targets
       location: https://github.com/openwrt/targets.git
       branch: master
+    - method: src-git
+      name: routing
+      location: https://github.com/openwrt-routing/packages.git
+      branch: openwrt-18.06
```
Comment (hispanico) [inline defaults/main.yml]: Are you sure this feed is needed as default?

File: defaults/main.yml [hunk 2]
```diff
@@ -60,6 +64,9 @@ openwisp2fw_source_additional_packages:
     - iputils-ping
     - ip-full
     - wget
+    - oonf-olsrd2
+    - luci
+    - luci-ssl
```
Comment (hispanico) [inline defaults/main.yml]: These are custom packages. We shoudn't need as default.

**Review by hispanico — CHANGES_REQUESTED** [review]
See my inline comments.

File: defaults/main.yml (hunk 1 above)
Comment (nemesifier) [reply defaults/main.yml]: not needed

File: defaults/main.yml (hunk 2 above)
Comment (nemesifier) [reply defaults/main.yml]: exactly, we don't need these additions in this patch

**Review by nemesifier — CHANGES_REQUESTED** [review]
You did the pull request title well but not the commit message, update the commit message to:

```
[main.yml] Updated to openwrt-18.06 #16

Closes #16 
```

**Comment by EdgeKing810** [comment]
image

Sorry, I was basing myself on the links provided on the links provided here: https://github.com/openwisp/ansible-openwisp2-imagegenerator/issues/16 

I did what you asked me to do in my last commit. Is there still anything more that needs to be done?

Closes #16

**Comment by hispanico** [comment]
No worries.
Just a last step, could you squash and rename your commits as @nemesisdesign suggest?
```
[main.yml] Updated to openwrt-18.06
```

**Comment by EdgeKing810** [comment]
And... it's done. Should I squash all commits for future tasks too as well now?

**Review by nemesifier — APPROVED** [review]

**Pull request merged**